docker/Dockerfile

```diff
@@ -81,7 +81,6 @@ RUN install2.r --error \
   && rm -rf /tmp/downloaded_packages/ /tmp/*.rds
 
 # Fix umask problems with RStudio in singularity
-RUN echo "server-set-umask=0" >> /etc/rstudio/rserver.conf
```

Comment:
This was the offending line. It tells RStudio not to set its own umask for users. But then, rather than use the user's own umask, it inherits the system default. And since I told it not to set umasks, it doesn't honor the next line!